src/stactools/sentinel2/stac.py

```diff
@@ -80,6 +81,7 @@ class Metadata:
     epsg: int
     proj_bbox: List[float]
     resolution_to_shape: Dict[int, Tuple[int, int]]
+    processing_baseline: str
```

Comment:
Can we enum this? Or are there a lot of baselines?

Reply:
There are enough that they would be hard to enumerate, and we want to future proof this for 04.01, etc.